lib/drivers/onetouch/oneTouchVerio.js

```diff
@@ -16,9 +16,11 @@
  */
 
 import fs from 'fs';
-import { assign, clone, invert } from 'lodash';
+import { assign, clone, invert, trimEnd } from 'lodash';
 // eslint-disable-next-line
```

Comment:
Why are the eslint-disables required for the following 2 modules?

Reply:
Otherwise we get a `Missing file extension for "drivelist" (import/extensions)` linting error. I'm guessing this is because they are native modules.

I'm going to add the `ignorePackages` option to `.eslintrc` so that we don't need the eslint-disables.